queue_services/business-digital-credentials/devops/vaults.gcp.env

```diff
@@ -15,4 +15,8 @@ BUSINESS_SCHEMA_NAME="op://bbdc/$APP_ENV/aca-py/BUSINESS_SCHEMA_NAME"
 BUSINESS_SCHEMA_VERSION="op://bbdc/$APP_ENV/aca-py/BUSINESS_SCHEMA_VERSION"
 BUSINESS_SCHEMA_ID="op://bbdc/$APP_ENV/aca-py/BUSINESS_SCHEMA_ID"
 BUSINESS_CRED_DEF_ID="op://bbdc/$APP_ENV/aca-py/BUSINESS_CRED_DEF_ID"
-WALLET_CRED_DEF_ID="op://bbdc/$APP_ENV/aca-py/WALLET_CRED_DEF_ID"
\ No newline at end of file
+WALLET_CRED_DEF_ID="op://bbdc/$APP_ENV/aca-py/WALLET_CRED_DEF_ID"
+
+# GCP
+SUB_AUDIENCE="op://gcp-queue/$APP_ENV/business/BUSINESS_DIGITAL_CREDENTIALS_SUB_AUDIENCE"
+SUB_SERVICE_ACCOUNT="op://gcp-queue/$APP_ENV/business/BUSINESS_SERVICE_ACCOUNT"
```

Comment:
Business Service Account doesn't exist in 1Pass.

<img width="273" height="307" alt="image" src="https://github.com/user-attachments/assets/3ae4a142-ca9f-41ff-bb14-57147e84f51e" />

Reply:
@bolyachevets showed me this screenshot:
<img width="1000" height="396" alt="image" src="https://github.com/user-attachments/assets/723c4d58-4aca-4a40-ac61-429086761883" />

The other queue-services have `op://gcp-queue/$APP_ENV/business/BUSINESS_SERVICE_ACCOUNT` and I can see that pulling in in the yamls for those so think it must be mapping?

Reply:
Is that going to be the same prefix in Test and Prod?